themes/src/main/resources/theme/keycloak/login/theme.properties

```diff
@@ -2,7 +2,7 @@ parent=base
 import=common/keycloak
 
 styles=css/login.css
-stylesCommon=node_modules/patternfly/dist/css/patternfly.min.css node_modules/patternfly/dist/css/patternfly-additions.min.css lib/zocial/zocial.css
+stylesCommon=node_modules/patternfly/dist/css/patternfly.min.css web_modules/@patternfly/react-core/dist/styles/app.css web_modules/@patternfly/react-core/dist/styles/base.css node_modules/patternfly/dist/css/patternfly-additions.min.css lib/patternfly4/tile.css
```

Comment:
I have some issues with margin and padding. If you add the `css` file in this order, like below, it should resolve that.
`
base.css, app.css, patternfly.min.css, patternfly-additions.min.css tile.css
`

Reply:
@mabartos What do you mean? I haven't noticed any issues. :)

Reply:
@vmuzikar (cc: @mabartos) me neither, anyway I change the order as suggested.